sceptre/plan/actions.py

```diff
@@ -465,8 +465,24 @@ def create_change_set(self, change_set_name):
                 {"Key": str(k), "Value": str(v)} for k, v in self.stack.tags.items()
             ],
         }
+
         create_change_set_kwargs.update(self.stack.template.get_boto_call_parameter())
         create_change_set_kwargs.update(self._get_role_arn())
+
+        try:
+            self._create_change_set(change_set_name, create_change_set_kwargs)
+        except Exception as err:
```

Comment:
can we add unit tests in `tests/test_actions.py` to verify these exceptions?

Reply:
@zaro0508 On looking at this carefully, we do not have any examples of asserting about exceptions in that file. Nor could I easily figure out how to add any. Noting that the behaviour added in this PR is tested by the integration tests, I would be inclined to not add unit tests here. Or, failing that, do you have any idea how to write the assertions?

Reply:
ok, integration tests will suffice